packages/scxa-cell-type-wheel-experiment-heatmap/src/CellTypeWheelExperimentHeatmap.js

```diff
@@ -51,13 +51,20 @@ function CellTypeWheelExperimentHeatmap (props) {
       />
       <div className={`row-expanded small-12 columns`}>
         <div className={`small-12 medium-6 columns`} aria-label={`Cell type wheel`}>
-          <CellTypeWheelFetchLoader
-            host={props.host}
-            resource={URI(props.searchTerm, props.cellTypeWheelResource).toString()}
-            fulfilledPayloadProvider={ cellTypeWheelData => ({ data: cellTypeWheelData }) }
-            searchTerm={props.searchTerm}
-            onCellTypeWheelClick={onCellTypeWheelClick}
-          />
+          {props.searchTerm.length > 0 ?
```

Comment:
In JavaScript an empty string is coerced to `false` in a Boolean expression, so you can do:
```js
{props.searchTerm ? ...}
```

You can go the extra mile:
```js
{ props.searchTerm.trim() ? ...}
```

Reply:
Done, used the 2nd.